src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/QuestionsAndAnswersDisplay/QuestionsAndAnswersDisplay.tsx

```diff
@@ -73,8 +73,8 @@ const QuestionsAndAnswersDisplay = (
                             : props.questionAndAnswer.answers?.id
                     }
                 >
-                    {item.valueBoolean && <p>Sant</p>}
-                    {item.valueBoolean === false && <p>Usant</p>}
+                    {item.valueBoolean && <span>Sant</span>}
+                    {item.valueBoolean === false && <span>Usant</span>}
```

Comment:
Any reason these elements are spans and not p, considering the content is just text?

Reply:
Changed to div and p